corehq/apps/export/export.py

```diff
@@ -356,26 +357,70 @@ def _write_export_instance(writer, export_instance, documents, progress_tracker=
     if progress_tracker:
         DownloadBase.set_progress(progress_tracker, 0, documents.count)
 
-    start = int(time.time() * 1000)
+    start = _time_in_milliseconds()
+    total_bytes = 0
+    total_rows = 0
+
     for row_number, doc in enumerate(documents):
+        doc_bytes = sys.getsizeof(doc)
+        total_bytes += doc_bytes
         for table in export_instance.selected_tables:
+            compute_start = _time_in_milliseconds()
             rows = table.get_rows(
                 doc,
                 row_number,
                 split_columns=export_instance.split_multiselects,
                 transform_dates=export_instance.transform_dates,
             )
+            compute_end = _time_in_milliseconds()
+            _record_datadog_export_compute_rows(compute_start, compute_end, doc_bytes, len(rows))
+
+            write_start = _time_in_milliseconds()
             for row in rows:
                 # It might be bad to write one row at a time when you can do more (from a performance perspective)
                 # Regardless, we should handle the batching of rows in the _Writer class, not here.
                 writer.write(table, row)
+            write_end = _time_in_milliseconds()
+
+            _record_datadog_export_write_rows(write_start, write_end, doc_bytes, len(rows))
```

Comment:
probably not, originally thought it might be nice to have the histogram stats on a row level for better accuracy, but thinking about it again i actually don't think there will be a lot of variance in these timings for one export